klaytn-tx-latency-measurement/sendtx_klaytn.js

```diff
@@ -13,6 +13,10 @@ const CoinGeckoClient = new CoinGecko();
 async function uploadToS3(data) {
     const s3 = new AWS.S3();
 
+    if(process.env.S3_BUCKET === "") {
```

Comment:
Can we put this part before `const s3 = new AWS.S3();`  like solana code?

Reply:
Okay. Will do.